Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift

```diff
@@ -1102,6 +1102,29 @@ public class ParseDiagnosticsGenerator: SyntaxAnyVisitor {
       moveFixIt: { MoveTokensAfterFixIt(movedTokens: $0, after: .equal) },
       removeRedundantFixIt: { RemoveRedundantFixIt(removeTokens: $0) }
     )
+
+    if let attributes = node.attributes,
+      attributes.isMissingAllTokens,
+      let unexpected = node.unexpectedBetweenBindingKeywordAndBindings
```

Comment:
I think we should also check that `unexpected` only contains a `AttributeListSyntax`. Otherwise we might emit this diagnostic if we just recover to the bindings and stuff any other node into `unexpectedBetweenBindingKeywordAndBindings`. I.e. I’d add

```suggestion
      let unexpected = node.unexpectedBetweenBindingKeywordAndBindings,
      unexpected.only?.is(AttributeListSyntax.self) ?? false
```